llvm/test/CodeGen/AArch64/cfi-annotation.ll

```diff
@@ -0,0 +1,114 @@
+; RUN: opt %s -o %t.bc
+; RUN: opt -passes=lowertypetests %t.bc -o %t.o
+
+; REM: Find the `llvm.global.annotations` symbol in `%t.*.ll` and verify
+; REM: that no function annotation references CFI jump table entry.
+
+; RUN: llvm-dis %t.o -o - | FileCheck %s --check-prefix=CHECK-bar
+; CHECK-bar: {{llvm.global.annotations = .*bar, }}
+
+; RUN: llvm-dis %t.o -o - | FileCheck %s --check-prefix=CHECK-foo
+; CHECK-foo: {{llvm.global.annotations = .*foo, }}
```

Comment:
Why do you need three different FileCheck invocations here? You should be able to match the whole llvm.global.annotations in one invocation.

The standard invocation is `RUN: opt -S -passes=lowertypetests < %s | FileCheck %s`.

Reply:
I need at least two invocations to FileCheck, one for positive check and the other for negative check.